import_map.json

```diff
@@ -1,15 +1,18 @@
 {
   "imports": {
     "@/": "./",
-    "$fresh/": "https://deno.land/x/fresh@1.1.4/",
+    "$fresh/": "https://deno.land/x/fresh@1.1.5/",
     "$gfm": "https://deno.land/x/gfm@0.2.1/mod.ts",
     "preact": "https://esm.sh/preact@10.13.2",
     "preact/": "https://esm.sh/preact@10.13.2/",
     "preact-render-to-string": "https://esm.sh/*preact-render-to-string@5.2.6",
     "@preact/signals": "https://esm.sh/*@preact/signals@1.1.3",
     "@preact/signals-core": "https://esm.sh/*@preact/signals-core@1.2.3",
-    "twind": "https://esm.sh/twind@0.16.19",
-    "twind/": "https://esm.sh/twind@0.16.19/",
+    "twind": "https://esm.sh/@twind/core@1.1.3",
+    "twind-preset-autoprefix": "https://esm.sh/@twind/preset-autoprefix@1.0.7",
+    "twind-preset-tailwind": "https://esm.sh/@twind/preset-tailwind@1.1.4/base",
+    "twind-preset-tailwind-colors": "https://esm.sh/@twind/preset-tailwind@1.1.4/colors",
+    "twind_fresh_plugin/": "https://deno.land/x/fresh_twindv1_plugin@v1.3.0/",
```

Comment:
could you use this instead?

https://github.com/denoland/fresh/blob/main/plugins/twindv1.ts